lib/wakes/redirect_mapper.rb

```diff
@@ -33,11 +35,11 @@ def resources_affected
     end
 
     def target_location
-      @target_location ||= Wakes::Location.find_by(:path => target)
+      @target_location ||= Wakes::Location.find_by(:host => host(target_path), :path => path(target_path))
```

Comment:
@benhutton You may recall that I had @target_host and @source_host variables in the last check-in. Because we are now running into Rubocop class length restrictions on `RedirectMapper`, I decided to condense these into two variables, since the other two didn't add much to readability. 

I also had @source_path_or_url before pushing the commits, but ran into line length issues (which couldn't be resolved without introducing class length issues).

Much of the stuff you're pointing out today is monkeying I did consciously to fit within the style guide. Let me relax the class length cop and then I think we'll be on the same page.